### PR TITLE
New device: Rohde & Schwarz Sx800

### PR DESCRIPTION
**Pull request by TheGreatDoc**
New device vendor Rohde & Schwarz.

In this case is for Sx800 but it should be valid for any of their Exciters products.

I had to use php for frequency due to index being a letter (a) translated to 1 in num oid.

I couldnt run ./lnms dev:check but I think it should work at 1st attempt 🤞 

DO NOT DELETE THE UNDERLYING TEXT

#### Please note

> Please read this information carefully. You can run `./lnms dev:check` to check your code before submitting.

- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)
- [x] If my Pull Request does some changes/fixes/enhancements in the WebUI, I have inserted a screenshot of it.

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

File: includes/definitions/rs.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+text: Rohde & Schwarz
+over:
+    - { graph: device_power, text: Output Power }
+type: network
+mib_dir: rs
+discovery:
+    -
+        sysObjectID:
+            - .1.3.6.1.4.1.2566.127.1.2.
+discovery_modules:
```
Comment (murrant) [inline includes/definitions/rs.yaml:11]: This line can be removed ;)

File: includes/discovery/sensors/frequency/rs.inc.php [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?php
+
+$oids = snmpwalk_cache_oid($device, 'cmdExcFrequency', [], 'RS-XX8000-DVB-TX-MIB');
```
Comment (murrant) [inline includes/discovery/sensors/frequency/rs.inc.php:3]: Any reason you couldn't do this in yaml?

Comment (murrant) [reply includes/discovery/sensors/frequency/rs.inc.php:3]: is $id a string?

Comment (TheGreatDoc) [reply includes/discovery/sensors/frequency/rs.inc.php:3]: Yes, its in the description of the PR ;)

I had to use php for frequency due to index being a letter (a) translated to 1 in num oid.

File: includes/definitions/rs.yaml (hunk 1 above)
Comment (TheGreatDoc) [reply includes/definitions/rs.yaml:11]: lol, missed that one.

**Review by Jellyfrog — APPROVED** [review]

**Comment by murrant** [comment]
This pull request has been mentioned on **LibreNMS Community**. There might be relevant details there:

https://community.librenms.org/t/v1-69-release-changelog-october-2020/13837/1

**Pull request merged**